src/draw/control/Draw.RemoveLastPoint.js

```diff
@@ -0,0 +1,15 @@
+L.Draw.RemoveLastPoint = L.ToolbarAction.extend({
+	options: {
+		toolbarIcon: { html: 'Delete last point' }
```

Comment:
Good catch! I moved all of these strings into the `L.drawLocal` object in the new [`leaflet-toolbar-plugin-patches`](https://github.com/manleyjster/Leaflet.draw/commit/113014aa17a8805d130737dde29728ecabe361fa) branch.